Python/MAD_LIBS_BOT/readme.md

```diff
@@ -0,0 +1,37 @@
+# Discord Bot
+We all use discord every day, but do we have fun with discord?
+This bot will provide you the fun you need ,its filled with funny phrases with your given inputs
+
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/)
+
+# Features
+
+This is a discord bot that takes inputs from the user and gives you funny phrases which will make your day
+It has timeout,always-on features too
+
+## Steps to try this
+
+1. Clone this repository
+
+```
+
+git clone clone_path
+
+```
+## How to use it:
+
+````python
+# install python3 and python3-pip only if they are not installed
+
+# install the requirements
+$ pip3 install discord.py
+$ pip3 install flask
+````
+````
+   python
+$ python3 main.py
+````
+
+# output
+
+<img src="output.png">
```

Comment:
Refer to this [link](https://github.com/HarshCasper/Rotten-Scripts/blob/master/CONTRIBUTING.md#note--do-not-add-images-rather-) and add links to the images.
Write it as `![Image](Imgur_link)]`